data-prepper-plugin-framework/src/main/java/org/opensearch/dataprepper/plugin/PluginArgumentsContext.java

```diff
@@ -7,4 +7,8 @@
 
 interface PluginArgumentsContext {
     Object[] createArguments(final Class<?>[] parameterTypes, final Object ... args);
+
+    default Object getArgument(int index) {
```

Comment:
Do we need this change?

Reply:
@dlvenable I couldn't another way to get the configuration of a plugin. Let me know if you can think of another way

Reply:
We can keep this. The interface is package protected anyway, so we can change it later if needed.